install_config/install/deploy_router.adoc

```diff
@@ -269,9 +279,89 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be used to distribute routing load in the cluster. Because
+each router uses ports 80 (http) and 443 (https) the routers are placed on nodes
+in the cluster that are not already using those ports.
+
+Each router handles a set of routes that are tagged using labels. Router sharding
+uses ROUTE_LABELS. A project's (namespace) routes can be associated with a router
+using NAMESPACE_LABELS.
+
+
+[[using-namespace-labels]]
+=== Using NAMESPACE_LABELS
+
+Each project has its own namespace. The routes for a project can be handled by a
+selected router by using NAMESPACE_LABELS. The router is given a label and each
+project that uses the router is also given the same label.
+
+In order for this to work the router service account associated with the router
+needs cluster-reader permission.
+The same service account can be used in creating multiple routers.
+
+Add cluster-reader to the router service account, in this case the default route:
+====
+----
+$ oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:default:router
+----
+====
+Now create and label the router:
+====
+----
+$ oadm router ...
+$ oc env dc/router NAMESPACE_LABELS="router=r1"
+----
+====
+Each project has a namespace and that namespace can be labeled with the
```

Comment:
We already talked about the relationship between namespaces and projects, let's just stick to namespaces here.